mvds.md

```diff
@@ -48,20 +48,21 @@ Payloads are implemented using [protocol buffers v3](https://developers.google.c
 syntax = "proto3";
 
 message Payload {
-  repeated bytes acks = 1;
-  repeated bytes offers = 2;
-  repeated bytes requests = 3;
-  repeated Message messages = 4;
+  repeated bytes acks = 5001;
+  repeated bytes offers = 5002;
+  repeated bytes requests = 5003;
+  repeated Message messages = 5004;
 }
 
 message Message {
-  bytes group_id = 1;
-  int64 timestamp = 2;
-  bytes body = 3;
+  bytes group_id = 6001;
+  int64 timestamp = 6002;
+  bytes body = 6003;
 }
-
 ```
 
+*The payload field numbers are kept more "unique" to ensure no overlap with other protocol buffers.*
```

Comment:
which other buffers though? and why 5 and 6? are 1 2 3 4 taken or reserved? what about future 7 and 8? this will be confusing in the future if we don't write this down explicitly imo

Reply:
Those numbers were decided by @cammellos, maybe he can chime in.